easybuild/easyconfigs/k/Keras/Keras-2.0.4-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'Keras'
+version = '2.0.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://keras.io/'
+description = """Keras is a minimalist, highly modular neural networks library, written in Python and
+capable of running on top of either TensorFlow or Theano."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('Theano', '0.9.0', versionsuffix),
+    ('h5py', '2.7.0', '%(versionsuffix)s'),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+# it defaults to Tensorflow
+modextravars = {'KERAS_BACKEND': 'theano'}
```

Comment:
@wpoely86 please also do this in the existing easyconfig for Keras (just for the v2.0.4 one is fine imho)?